providers/hns/hns_roce_u.h

```diff
@@ -253,6 +254,32 @@ static inline unsigned long align(unsigned long val, unsigned long align)
 	return (val + align - 1) & ~(align - 1);
 }
 
+static inline int hr_ilog2(int n)
```

Comment:
Use ccan/ilog.h and remove mlx5_ilog2() in favour of ccan too.

Reply:
Hi Leon, we have pushed a v2 patch-set to fix your comments and remove the patch that needs-kernel-patch, could you please review again? Thank you :)